src/index.css

```diff
@@ -1,138 +1,152 @@
-@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300..700&display=swap');
+@import url("https://fonts.googleapis.com/css2?family=Quicksand:wght@300..700&display=swap");
 
 * {
-    box-sizing: border-box;
-    list-style: none;
-    margin: 0;
-    outline: 0;
-    padding: 0;
-    text-decoration: none;
+  box-sizing: border-box;
+  list-style: none;
+  margin: 0;
+  outline: 0;
```

Comment:
This disables focus outlines for keyboard users, harming accessibility. Outline:0